requirements.txt

```diff
@@ -55,6 +56,7 @@ pandas==0.23.4
 parsedatetime==2.0.0
 pathlib2==2.3.0
 polyline==1.3.2
+PTable==0.9.2
```

Comment:
It doesn't have a lot of activity but then it wouldn't need it - it just pretty prints data and it works :) What do you think? This seems the best way to print a table because `prettytable` hasn't had a commit since 2014 and is on google code: https://code.google.com/archive/p/prettytable/ `PTable` is a fork of `prettytable`.

Hmmmm... now that I look at it, https://pypi.org/project/tabulate/ looks like a better choice. It was just recently released. Let me try that.

Reply:
I've replaced this with `tabulate` and it works well.